bin/main.rb

```diff
@@ -1,2 +1,110 @@
+player_two_name = gets.chomp
+while player_two_name == ''
+  puts 'Hey, please, tell us your name :)'
+  player_two_name = gets.chomp
+end
+player_two = Player.new(player_two_name, 'O')
+
+puts "Huuummmm #{player_two.name}, I was pretty aware that your name would be also beautiful!"
+puts "As #{player_one.name} is already using the 'X' character, you are going to be the 'O' character, nice? (Press Enter)"
+gets.chomp
+
+puts "#{player_one.name} and #{player_two.name}, please promise me you won't forget your characters? (Press Enter)"
+gets.chomp
+
+puts "Ok #{player_one.name} and #{player_two.name}, lets to the nuts and bolts!"
+puts 'Here is the board of your game, with the respective number for each position, ok?'
+plays = [1, 2, 3, 4, 5, 6, 7, 8, 9]
+
+class Gameboard
+  def print_board(plays)
+    puts " #{plays[0]} | #{plays[1]} | #{plays[2]}"
+    puts " #{plays[3]} | #{plays[4]} | #{plays[5]}"
+    puts " #{plays[6]} | #{plays[7]} | #{plays[8]}"
+  end
+end
+
+gameboard_instance = Gameboard.new
+puts gameboard_instance.print_board(plays)
+
+class Gameplay
+  def catch_play(plays)
+    play = play.to_i
+    while plays[play - 1] == 'X' || plays[play - 1] == 'O' || !(play.to_i.positive? && play.to_i <= 9)
+      play = gets.chomp.to_i
+      if plays[play - 1] == 'X' || plays[play - 1] == 'O' || !(play.to_i.positive? && play.to_i <= 9)
+        puts 'Ops, it seems you did not type a number between 1-9 OR someone already choosen this slot before :/'
+      end
+    end
+    play
+  end
+end
+
+gameplay_instance = Gameplay.new
+
+call_for_play_phrases = []
+call_for_play_phrases << 'So now, PLAYER, could you please choose one of the slots in the board?'
+call_for_play_phrases << 'PLAYER, now it\'s your turn!'
+call_for_play_phrases << 'PLAYER, no more. Choose your slot please.'
+call_for_play_phrases << 'PLAYER, choose your slot and don\'t let your oponent to conquer the world!'
+call_for_play_phrases << 'PLAYER, breath and think twice your next slot, it can be not so easy as you think!'
+call_for_play_phrases << 'PLAYER, what are you waiting for choose your slot??'
+call_for_play_phrases << 'PLAYER, please choose your slot, we don\'t have all the time of the world!!'
+call_for_play_phrases << 'PLAYER, we are waiting for you...'
+call_for_play_phrases << 'PLAYER, please don\'t make us wait more'
+
+comment_play_phrases = []
+comment_play_phrases << 'Huummm... wise fox play!'
+comment_play_phrases << 'WoW! I\'m noticing this will be a giant\'s game!'
+comment_play_phrases << 'Huummm!'
+comment_play_phrases << 'ok'
+comment_play_phrases << 'Hey, how many years did you study this game?'
+comment_play_phrases << 'Couldn\'t you think something better than this??'
+comment_play_phrases << 'Well I don\'t like this play, but anyway...'
+comment_play_phrases << 'You never played this before, right?'
+comment_play_phrases << '... so amateur'
+comment_play_phrases << 'Wow!!! Now I see your potential!'
+
+9.times do |i|
```

Comment:
- [ ] For this, the while gives you more control because you can automatically terminate the game even if it is before the 9th times (without to use the break). As it was required, I think you could change it, even if this works. The while is a little bit more accurate, and it won't be that much trouble.